docs/docs/References/Plugin_Operations.md

```diff
@@ -823,6 +824,50 @@ def reconfigure(virtual_source, repository, source_config, snapshot):
 }
 ```
 
+## Virtual Source Cleanup
+
+Intended to allow a final cleanup during a delete operation, unlike unconfigure which can be used to signal a temporary dissassociation with a database.
+
+Cleanup is called during the delete flow after unconfigure.
```

Comment:
Maybe "cleanup is called as the final step of the delete workflow".  I only suggest this because there are times when the delete workflow does not call unconfigure.